components/calendar/src/cal/japanese.rs

```diff
@@ -17,7 +17,7 @@ use tinystr::tinystr;
 /// The months and days are identical to that of the Gregorian calendar, however the years are counted
 /// differently using the Japanese era system.
 ///
-/// This calendar only contains eras after Meiji, for all historical eras, check out [`JapaneseExtended`].
+/// The [Japanese calendar] was taken into use starting from 6 Meiji (1873 ISO/Gregorian), so years before 6 Meiji are proleptic. This implementation uses the Meiji era proleptically back to and including 1868-10-23 ISO and uses the Gregorian eras for 1868-10-22 ISO and earlier dates. For a variant that uses approximations of historical Japanese eras proleptically, check out [`JapaneseExtended`].
```

Comment:
"from 6 Meiji" - please give the exact date here instead of below

Reply:
Furthermore, since the implementation is proleptic to the start of the Meiji era anyway, the exact actual date of use isn't that important for the docs.

Reply:
I guess https://en.wikipedia.org/wiki/Adoption_of_the_Gregorian_calendar#Countries_that_used_lunisolar_calendars confirms January 1st.

Reply:
Adjusted the text.